.github/workflows/sdk-release-manual.yml

```diff
@@ -107,23 +107,23 @@ jobs:
           SALT: ${{ secrets.SALT }}
         run: |
           pnpm run build:contracts
-          pnpm --filter @nexeraid/sig-gating-contracts-sdk build
-          pnpm --filter @nexeraid/tezos-sig-gating-contracts-sdk build
+          pnpm --filter @compilot/sig-gating-contracts-sdk build
+          pnpm --filter @compilot/tezos-sig-gating-contracts-sdk build
 
 #      - name: Publish to private NPM
 #        run: |
-#          pnpm --filter @nexeraid/sig-gating-contracts publish --no-git-checks --registry https://npm.pkg.github.com/ --tag ${{ steps.tag_release.outputs.release_tag }}
-#          pnpm --filter @nexeraid/sig-gating-contracts-sdk publish --no-git-checks --registry https://npm.pkg.github.com/ --tag ${{ steps.tag_release.outputs.release_tag }}
-#          pnpm --filter @nexeraid/tezos-sig-gating-contracts-sdk publish --no-git-checks --registry https://npm.pkg.github.com/ --tag ${{ steps.tag_release.outputs.release_tag }}
+#          pnpm --filter @compilot/sig-gating-contracts publish --no-git-checks --registry https://npm.pkg.github.com/ --tag ${{ steps.tag_release.outputs.release_tag }}
+#          pnpm --filter @compilot/sig-gating-contracts-sdk publish --no-git-checks --registry https://npm.pkg.github.com/ --tag ${{ steps.tag_release.outputs.release_tag }}
+#          pnpm --filter @compilot/tezos-sig-gating-contracts-sdk publish --no-git-checks --registry https://npm.pkg.github.com/ --tag ${{ steps.tag_release.outputs.release_tag }}
 
       - name: Publish to public NPM
         env:
           NPMJS_COM_TOKEN: ${{ secrets.NPMJS_COM_TOKEN }}
```

Comment:
we need to use the new token, same in the sdk-release.yml

`secrets.COMPILOT_NPMJS_COM_TOKEN`